packages/connectivity/src/scp-cf/get-protocol.ts

```diff
@@ -23,7 +23,7 @@ export function getProtocolOrDefault(destination: Destination): Protocol {
     );
     return Protocol.HTTPS;
   }
-  const casted = Protocol.of(protocol[0]);
+  const casted = ProtocolNamespace.of(protocol[0]);
```

Comment:
[req] I guess the idea behind the namespace was to give the `of`method some context so that it is clear what it creates. I guess if this leads to some name collision with the `Protocol` object I would just create a method `createProtocol(str:string)` or `getProtocal` and get rid of the namespace.

Reply:
[q] There was also the issue with the mail client package name. Is this solved?

Reply:
> [q] There was also the issue with the mail client package name. Is this solved?

yes. it comes from missing JSDoc annotations for typedoc in the index.ts. It needs the annotations:
```
/**
 * [[include:mail-client/README.md]]
 * @packageDocumentation
 * @module @sap-cloud-sdk/mail-client
 */
```
[Actually, this has already been added](https://github.com/SAP/cloud-sdk-js/commit/838aaef7d280eb002d9abb563f483104c0a86da4) but it is not on docs repo. The incorrect module name will be updated correctly.

Reply:
> [req] I guess the idea behind the namespace was to give the `of`method some context so that it is clear what it creates. I guess if this leads to some name collision with the `Protocol` object I would just create a method `createProtocol(str:string)` or `getProtocal` and get rid of the namespace.

I made a `getProtocol()` to get rid of the namespace. It also works fine.